.eslintrc.js

```diff
@@ -1,3 +1,5 @@
+// this module.exports is needed for eslint
```

Comment:
I encountered this myself the other day.  The solution is to add `node: true` to the `env`object.